index.html

```diff
@@ -113,7 +133,7 @@
           <div class="nav-login">
             <a
               class="btn-login"
-              href="https://github.com/login/oauth/authorize?client_id=23c78f66ab7964e5ef97"
+              href="https://github.com/login/oauth/authorize?client_id=8b97fd58a86f1d06b0e2"
```

Comment:
why are we hard coding the client id, is it a good practice...
can we please find a better solution for it?